operations/host_spawn.go

```diff
@@ -553,17 +555,28 @@ func hostModifyVolume() cli.Command {
 				Name:  joinFlagNames(sizeFlag, "s"),
 				Usage: "set new volume `SIZE` in GiB",
 			},
+			cli.DurationFlag{
+				Name:  extendFlag,
+				Usage: "extend volume's expiration",
```

Comment:
I see that you're using a duration flag but I think we should try to keep the host and volume routes as similar as possible, so if you want to use "duration" instead of "hours" then I think host should also use duration.

If this isn't possible let me know, it was just a thought.

Reply:
brought volumes in line with hosts (both take an hours int arg)